src/modules/party/dto/get-parties-by-search-text-query.dto.ts

```diff
@@ -1,7 +1,7 @@
 import { ApiProperty } from '@nestjs/swagger';
 import { Length } from 'class-validator';
 
-export class PartiesQueryDto {
+export class GetPartiesBySearchTextQuery {
```

Comment:
Should we keep `Dto` at the end here?

Reply:
Possibly, although I notice that this is the only class name (at least in the `party` module) containing `Dto`, the others are of the form e.g. `GetPartiesBySearchTextResponse`

Reply:
Okay, I agree with removing it. Thanks